ooyala_player/ooyala_player.py

```diff
@@ -316,7 +316,7 @@ def api_key_3play_with_default_setting(self):
 
         settings_service = self.runtime.service(self, 'settings')
         try:
-            return settings_service.get('ENV_TOKENS')['XBLOCK_OOYALA_3PLAY_API']
+            return settings_service.get_settings_bucket(self, {'3PLAY_API_KEY': self.api_key_3play})['3PLAY_API_KEY']
         except (AttributeError, KeyError):
             return self.api_key_3play
```

Comment:
I think the default return value here (in `except`) and on the changed line 319 should just be `None` because line 314 has already established that `self.api_key_3play` is not set.